packages/integrations/mdx/src/index.ts

```diff
@@ -0,0 +1,38 @@
+import type { AstroIntegration } from 'astro';
+import mdxPlugin from '@mdx-js/rollup';
+
+export default function mdx(): AstroIntegration {
+	return {
+			name: '@astrojs/mdx',
+			hooks: {
```

Comment:
Noticing we're using tabs over spaces in these new files (which I honestly prefer as a repo default!). Should we standardize?

Reply:
We should be standardized already! Prettier should enforce tabs.

Reply:
If that's not the case, @fredkbot will definitely enforce whatever the correct format is upon merge to `main`. 😉